### DISCUSSION TITLE
[FEATURE]: Find taxonomy by name

### DISCUSSION THREAD

**Discussion by xanhz** [etsy/open-api#750] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

I'm a seller on Etsy and I also have own web app for selling products. Now I want to sync my products into Etsy. In choosing taxonomy step, I see that Etsy only give me an API for get all taxonomy nodes: https://openapi.etsy.com/v3/application/buyer-taxonomy/nodes

### Describe the solution you'd like

But I want to search taxonomy to choose easier and I found that there is an API when I create a product directly on Etsy: https://www.etsy.com/api/v3/ajax/bespoke/public/search/seller-listing-category?q=123&limit=5
Can I call that API from my app? Does Etsy block me to call it if I request too many times?

**Comment by etsyachristensen** [comment]
@XanhZ We are moving this to a Discussion Idea to continue tracking. This may be a nice feature for future users.
